File: test_samples/universal/hello.go
package main

import "fmt"

type Person struct {
    Name string
    Age  int
}

func (p Person) Greet() {
    fmt.Printf("Hello, my name is %s\n", p.Name)
}

func main() {
    p := Person{Name: "Alice", Age: 30}
    p.Greet()
}

File: test_samples/universal/hello.rb
module Geometry
  class Rectangle
    attr_reader :width, :height
    
    def initialize(width, height)
      @width = width
      @height = height
    end
    
    def area
      @width * @height
    end
  end
end

def greet(name)
  puts "Hello, #{name}!"
end
